src/Layout/PageBody.tsx

```diff
@@ -9,6 +9,7 @@ export interface PageBodyProps {
 
 const PageBody = styled.main<PageBodyProps>`
   margin: ${fromTheme('ws', 'small')};
+  padding-bottom: 40px;
```

Comment:
I've had a look at this on info.seas.harvard.edu and it seems as though the height of the footer is obscuring the bottom of the `PageBody` component
![image](https://github.com/seas-computing/mark-one/assets/537684/741e8ba8-0370-48a3-9dcb-97cb946eb32d)

Given that, setting `padding: 40px;` on the `PageBody` component as you have here does indeed fix the issue

![image](https://github.com/seas-computing/mark-one/assets/537684/bc2990cd-b1e9-4322-9a2c-2e4de122c5ec)

However, if the footer is made taller, we're likely going to run into the same issue again.

![image](https://github.com/seas-computing/mark-one/assets/537684/9eb8a946-0996-4567-ba14-8588f8141974)

Given that - I wonder if it makes sense to instead have either have optional `padding` parameter(s)? that one can pass(as well as a `height` parameter we can pass to [`Footer.tsx`](https://github.com/seas-computing/mark-one/blob/develop/src/Layout/Footer.tsx#L31) at some point in the future)

Reply:
Thanks Robert, that's a really good point. I updated the `PageBody` component to have an optional parameter for `padding` to give the user more flexibility.

Reply:
Looks great. Merge at will.